python/ray/train/torch/torch_trainer.py

```diff
@@ -45,32 +46,33 @@ def train_loop_per_worker(config: Dict):
     Inside the ``train_loop_per_worker`` function, you can use any of the
     :ref:`Ray AIR session methods <air-session-ref>`.
 
-    .. code-block:: python
+    .. testcode::
 
         def train_loop_per_worker():
             # Report intermediate results for callbacks or logging and
             # checkpoint data.
+            #
             session.report(...)
 
-            # Returns dict of last saved checkpoint.
+            # Session returns dict of last saved checkpoint.
```

Comment:
Say `Get dict of last saved checkpoint.` here? `Session returns` is a little confusing I think :)

Reply:
Yes, Get x makes sense then returns, since it's an explicit method call to sessin.get_xxx